server/index.js

```diff
@@ -113,33 +115,42 @@ async function handleAlert(location, alertType) {
   }
 }
 
-app.post('/api/sensorEvent', Validator.body(['coreid', 'event']).exists(), async (request, response) => {
+app.post('/api/sensorEvent', Validator.body(['coreid', 'event', 'api_key']).exists(), async (request, response) => {
```

Comment:
Have you tried to see if this works on your real Sensor? If so, could you please update your test plan to include what tests you ran?